src/inspector/models/notation/notes/beams/beamsettingsmodel.cpp

```diff
@@ -194,11 +194,12 @@ void BeamSettingsModel::setBeamHeight(const qreal left, const qreal right)
 void BeamSettingsModel::updateFeatheringMode(const qreal left, const qreal right)
 {
     if (left != right) {
-        setFeatheringMode(left > right ? BeamTypes::FeatheringMode::FEATHERED_DECELERATE
-                          : BeamTypes::FeatheringMode::FEATHERED_ACCELERATE);
+        m_featheringMode = left > right ? BeamTypes::FeatheringMode::FEATHERED_DECELERATE
+                           : BeamTypes::FeatheringMode::FEATHERED_ACCELERATE;
     } else {
-        setFeatheringMode(BeamTypes::FeatheringMode::FEATHERING_NONE);
+        m_featheringMode = BeamTypes::FeatheringMode::FEATHERING_NONE;
     }
+    emit featheringModeChanged(m_featheringMode);
```

Comment:
Before emitting this signal, you should check if m_featheringMode has really changed

Reply:
done! 👍